pkg/segment/test/set_functions.go

```diff
@@ -16,9 +16,16 @@ package segment
 
 // Basic numeric constants that we define because the math package doesn't.
 // TODO(nlacasse): These should be Math.MaxInt64/MinInt64?
+// TODO: this is commented because the return result on my computer is obviously not an int
+
+//const (
+//	maxInt = int(^uint(0) >> 1)
+//	minInt = -maxInt - 1
+//)
+
 const (
-	maxInt = int(^uint(0) >> 1)
```

Comment:
These really should be correct. Let's figure out what's going on here.

Reply:
It was a weird result on my computer but it now it seems good so I reverted it.